cabal.project

```diff
@@ -17,6 +17,7 @@ packages:
   eras/shelley/test-suite
   eras/shelley-ma/impl
   eras/shelley-ma/test-suite
+  libs/cardano-ledger-api
```

Comment:
Was wondering if  we want to add it to the list [below](https://github.com/input-output-hk/cardano-ledger/pull/2995/commits/1be5a8b04a35deeb75661f7d8862f3a5e2adb752#diff-b8ed06757045fac949c89f2139a862498ad2b6d1f82c61a31e7c91d6cf0eaa70L125) which enumerates each package in order to set " -Werror"  , but then I clicked on the link explaining why it's done and the github issue is closed, so maybe the whole enumerating is not necessary at all.  I can check later if that's the case

Reply:
Yeah, you are right on both counts. We do need to add a line for that package. And yes it was fixed, but we need to upgrade to cabal-3.8 for a single top level `-Werror` to work. I've created a ticket for that: https://github.com/input-output-hk/cardano-ledger/issues/2998